From 96233a8382b9520293a48b08dc3b204a21205800 Mon Sep 17 00:00:00 2001
From: Wang Mingyu <wangmy@cn.fujitsu.com>
Date: Fri, 24 Apr 2020 00:53:19 +0900
Subject: [PATCH] Fix bug of free() with musl

This fixes build error with musl C library:
/ubinux-dev/ubinux001/contribution/build/tmp/work/armv7vet2hf-neon-poky-linux-musleabi/paho-mqtt-c/1.3.2-r0/git/src/Heap.h:55:24:
error: expected declaration specifiers or '...' before string constant
   55 | #define free(x) myfree(__FILE__, __LINE__, x)
      |                        ^~~~~~~~
/ubinux-dev/ubinux001/contribution/build/tmp/work/armv7vet2hf-neon-poky-linux-musleabi/paho-mqtt-c/1.3.2-r0/git/src/Heap.h:55:34:
error: expected declaration specifiers or '...' before numeric constant
   55 | #define free(x) myfree(__FILE__, __LINE__, x)
      |                                  ^~~~~~~~

Signed-off-by: Wang Mingyu <wangmy@cn.fujitsu.com>
---
 src/MQTTReasonCodes.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/MQTTReasonCodes.c b/src/MQTTReasonCodes.c
index 416eab5..479dbac 100644
--- a/src/MQTTReasonCodes.c
+++ b/src/MQTTReasonCodes.c
@@ -16,10 +16,10 @@
 
 #include "MQTTReasonCodes.h"
 
-#include "Heap.h"
 #include "StackTrace.h"
 
 #include <memory.h>
+#include "Heap.h"
 
 #define ARRAY_SIZE(a) (sizeof(a) / sizeof(a[0]))
 
-- 
2.17.1

